src/Conf.ml

```diff
@@ -844,6 +845,15 @@ module Formatting = struct
       (fun conf x -> {conf with margin= x})
       (fun conf -> conf.margin)
 
+  let margin_check =
+    let doc =
+      "Check that the formatted output does not exceed the margin."
```

Comment:
Could you say that it's not an error but a warning?

Reply:
done